content/kotlin/concepts/datetime/terms/encodeToString/encodeToString.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'encodeToString()'
+
+Description: 'Serializes data to a string representation using the Kotlin Serialization library.'
+
+Subjects:
+  -'Kotlin'
+  -'Serialization'
```

Comment:
The subjects should come from [here](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md)